src/androidTest/java/com/example/qian/cs446project/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.example.qian.cs446project;

```

Comment:
I think this is the domain name right now. I'm ok keeping it like this for the remainder of the project if you two are. @KeQiaoChen ?

Reply:
Sure.